tests/test_cli.py

```diff
@@ -148,8 +146,7 @@ def test_update(cruft_runner, cookiecutter_dir):
 
 
 def test_update_with_conflicts(cruft_runner, cookiecutter_dir):
-    with (cookiecutter_dir / "README.md").open("w") as f:
-        f.write("conflicts")
+    (cookiecutter_dir / "README.md").write_text("conflicts")
```

Comment:
Same here?

Let's try and limit the changes to the actual implementation in this PR and do the minor cosmetic updates in a separate one. This also applies to the other such changes that have been made.

Makes it easier to review and keeps the PR focused on 1 feature.

Reply:
I removed them.